packages/framework/src/types/import.types.ts

```diff
@@ -0,0 +1,34 @@
+export type ImportRequirement = {
```

Comment:
Used by the `checkDependencies` util and in places that need to list their peer dependencies for checking, such as the validators.

Reply:
We [declare all the public-facing exports explicitly](https://github.com/novuhq/novu/blob/next/packages/framework/src/index.ts), so this shouldn't be a problem.

Here is a screen showing that the `ImportRequirement` type isn't available publicly:
<img width="742" alt="image" src="https://github.com/user-attachments/assets/d21a7782-571f-4854-bf4a-e987a1555072">

Reply:
As a side note for completeness, it's preferable to keep all the types located in the `types` directory and use explicit exports to make specific APIs available to keep the folder structure well organized.